admins/README.md

```diff
@@ -0,0 +1,129 @@
+# Administrators
+
+## Table of Contents
+- [Components](#components)
+- [Architecture](#architecture)
+
+## Components
+The BonnyCI system is made up of the following components:
+  - [Zuul](#zuul)
```

Comment:
Should these be added to the `Table of Contents`? Since they are (sub-)sections below, I would think so.